packit_service/worker/helpers/build/babysit.py

```diff
@@ -483,6 +483,14 @@ def update_vm_image_build(build_id: int, build: "VMImageBuildTargetModel"):
         message = f"No response for VM Image Build {build_id}: {ex}"
         logger.error(message)
         status = VMImageBuildStatus.error
+    except Exception as ex:
+        message = (
+            f"There was an exception when getting status of the VM "
+            f"Image Build {build_id}: {ex}"
+        )
+        logger.error(message)
```

Comment:
I have a feeling this will still pop up in Sentry as it is _an error_ :eyes:

Reply:
yes it will, I thought it still may be beneficial to know about them :thinking: but I can change it to debug

Reply:
no problem if it's intentional :D